util/network/src/lib.rs

```diff
@@ -145,14 +148,61 @@ impl NonReservedPeerMode {
 	}
 }
 
+#[derive(Clone, Debug, PartialEq, Eq)]
+#[cfg_attr(feature = "ipc", binary)]
+pub struct IpFilter {
+    pub predefined: AllowIP,
+    pub custom_allow: Vec<IpNetwork>,
+    pub custom_block: Vec<IpNetwork>,
+} 
+
+impl IpFilter {
+    pub fn new() -> IpFilter {
```

Comment:
I'd name it `default`  or `none` or `empty` to show that it creates an empty filter